src/phoenix/trace/langchain/instrumentor.py

```diff
@@ -0,0 +1,38 @@
+# Instruments the OpenInferenceTracer for LangChain automatically
+from typing import Any, Optional
+
+from phoenix.trace.instrumentor import Instrumentor
+
+from .tracer import OpenInferenceTracer
+
+MINIMUM_LANGCHAIN_VERSION = "0.13.0"
```

Comment:
should this be used inside `instrument()`?

Reply:
Oh will nix